src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/offender_extra_info.json

```diff
@@ -154,6 +154,7 @@
   "status": "ACTIVE IN",
   "legalStatus": "IMMIGRATION_DETAINEE",
   "imprisonmentStatus": "DEPORT",
+  "receptionDate": "2021-09-06",
```

Comment:
This file is used by multiple test for pulling data back in different endpoints.  Data set should be the same for all.